### ISSUE TITLE
Error in staking with "Rebag" and "Move up" functions

### ISSUE THREAD

**Issue by wellfitness** [Zondax/ledger-polkadot#132]
I am trying to stake Polkadot using a linked Ledger account. Ledger is my stash account and I have another own Polkadot account as controller.

My problem is that I need to move within the exchange that has been assigned to me, but to execute the function I must sign with Ledger and it returns an error "Unexpected call index". The operation is not performed. The same thing happens if you try to perform a "Rebag" to change your bag.

The same thing happened to me in the previous exchange and I increased the stake, but the problem has been repeated after 3 days.
There are more users using Ledger with the same problem, could those features be added to make it compatible with the Polkadot Js extension?

Attached screenshots

Thanks for your help

![transaccion Ledger bolsa](https://user-images.githubusercontent.com/66517269/163791202-1279dd97-8d58-4092-8a17-892645a54217.PNG)
![bolsas polkadot](https://user-images.githubusercontent.com/66517269/163791203-17c4790e-3f12-4da4-a088-382f88630495.PNG)
![error ledger](https://user-images.githubusercontent.com/66517269/163791204-72c0430e-09f8-4e9a-b6e5-15c8ccd8bf16.PNG)

**Comment by ftheirs** [comment]
Hi @wellfitness! BagsList pallets is supported. Are you using a NanoS device? If so, you will need to install XL version or use a NanoX/NanoS+.
Just keep in mind that XL version requires more space and there might be no room for other apps in NanoS

**Comment by wellfitness** [comment]
> 

Yes, I use a Nano S.

If I try the XL version, then can I go back in case I can't add the Polkadot app?

**Comment by wellfitness** [comment]
Yes, I use a Nano S.

I have installed the XL version and indeed it works correctly.

I have no problem because I just want to use it for Polkadot. Thank you very much.

I take this opportunity to leave here the article with the steps I have followed to fix this:

https://support.polkadot.network/support/solutions/articles/65000179210-ledger-error-not-supported-or-unexpected-call-index-

Thanks for the help

![mover bolsa Ledger](https://user-images.githubusercontent.com/66517269/163818233-4daf570c-f232-44f3-a799-25cef8aa2c8a.PNG)
